packages/loader/driver-utils/src/adapters/compression/compressionTypes.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export enum SummaryCompressionAlgorithm {
+	None = 1,
+	LZ4 = 2,
+}
+export enum SummaryCompressionProcessor {
+	SummaryBlob = 1,
+}
+
+export interface ICompressionStorageConfig {
+	algorithm: SummaryCompressionAlgorithm;
+	minSizeToCompress: number;
+	processor: SummaryCompressionProcessor;
+}
```

Comment:
@justus-camp - thoughts on if we should merge compression config into a single interface?

Here is the op compression interface for reference:
```
export interface ICompressionRuntimeOptions {
	/**
	 * The minimum size the batch's payload must exceed before the batch's contents will be compressed.
	 */
	readonly minimumBatchSizeInBytes: number;

	/**
	 * The compression algorithm that will be used to compress the op.
	 */
	readonly compressionAlgorithm: CompressionAlgorithms;
}
```